### ISSUE TITLE
Different accuracy performance between the official botsort implementation and this repository implementation

### ISSUE THREAD

**Issue by amirmgh1375** [mikel-brostrom/yolo_tracking#980]
### Search before asking

- [X] I have searched the Yolo Tracking [issues](https://github.com/mikel-brostrom/yolo_tracking/issues) and found no similar bug report.

### Question

I have tested the main official implementation of the bot sort tracking repository.
When I ran this repository bot sort implementation I found that the performance between two implementations are so different. I have tested some hyper-parameters with some detection models, but I cannot reach the official repository.
Everything performs well but when occlusion occurs between two object, tracking IDs change.  

If any one has encountered this problem, please help me.

**Comment by mikel-brostrom** [comment]
Have you read the BoT-SORT paper? They use a custom YOLO-X trained on Crodwhuman:

> "For fair comparisons, we directly apply the publicly available detector of YOLOX [17], trained by [58] for MOT17, MOT20, and ablation study on MOT17."

and [58] states: 

> "The detector is YOLOX [24] with YOLOX-X as the backbone and COCO-pretrained mode l[36] as the initialized weights. For MOT17, the training schedule is 80 epochs on the combination of MOT17, CrowdHuman, Cityperson and ETHZ. For MOT20 and HiEve, we only add CrowdHuman as additional training data."

Unless you are using this model you won't see the same results.
